docs/topics/whatsnew-eap.md

```diff
@@ -1216,11 +1320,58 @@ fun writeBooksTo(writer: XMLWriter) {
 }
 ```
 
+### Common protected property AbstractMutableList.modCount
+
+The release makes the [`modCount`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/-abstract-mutable-list/mod-count.html)
```

Comment:
The release changes the `modCount` `protected` property of the `AbstractMutableList` interface to common.